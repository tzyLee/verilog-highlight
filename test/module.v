module _testbed_01$;
endmodule

module/**/test();
endmodule

module/**/ //
/**/ module_name
//
(/**/ //

) //
;
endmodule

module complex_ports ({c, d}, .e(f));
endmodule

module renamed_concat (.a({b, c}), f, .g(h[1]));
endmodule