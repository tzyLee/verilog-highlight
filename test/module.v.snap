>module _testbed_01$;
#^^^^^^ source.verilog meta.module.verilog storage.type.module.verilog
#      ^ source.verilog meta.module.verilog
#       ^^^^^^^^^^^^ source.verilog meta.module.verilog entity.name.type.module.verilog
#                   ^ source.verilog meta.module.verilog punctuation.section.module.begin.verilog
>endmodule
#^^^^^^^^^ source.verilog meta.module.verilog storage.type.module.verilog
>
>module/**/test();
#^^^^^^ source.verilog meta.module.verilog storage.type.module.verilog
#      ^^ source.verilog meta.module.verilog comment.block.verilog punctuation.definition.comment.begin.verilog
#        ^^ source.verilog meta.module.verilog comment.block.verilog punctuation.definition.comment.end.verilog
#          ^^^^ source.verilog meta.module.verilog entity.name.type.module.verilog
#              ^ source.verilog meta.module.verilog punctuation.definition.ports.begin.verilog
#               ^ source.verilog meta.module.verilog punctuation.definition.ports.end.verilog
#                ^ source.verilog meta.module.verilog punctuation.section.module.begin.verilog
>endmodule
#^^^^^^^^^ source.verilog meta.module.verilog storage.type.module.verilog
>
>module/**/ //
#^^^^^^ source.verilog meta.module.verilog storage.type.module.verilog
#      ^^ source.verilog meta.module.verilog comment.block.verilog punctuation.definition.comment.begin.verilog
#        ^^ source.verilog meta.module.verilog comment.block.verilog punctuation.definition.comment.end.verilog
#          ^ source.verilog meta.module.verilog
#           ^^ source.verilog meta.module.verilog comment.line.double-slash.verilog punctuation.definition.comment.verilog
>/**/ module_name
#^^ source.verilog meta.module.verilog comment.block.verilog punctuation.definition.comment.begin.verilog
#  ^^ source.verilog meta.module.verilog comment.block.verilog punctuation.definition.comment.end.verilog
#    ^ source.verilog meta.module.verilog
#     ^^^^^^^^^^^ source.verilog meta.module.verilog entity.name.type.module.verilog
>//
#^^ source.verilog meta.module.verilog comment.line.double-slash.verilog punctuation.definition.comment.verilog
>(/**/ //
#^ source.verilog meta.module.verilog punctuation.definition.ports.begin.verilog
# ^^ source.verilog meta.module.verilog comment.block.verilog punctuation.definition.comment.begin.verilog
#   ^^ source.verilog meta.module.verilog comment.block.verilog punctuation.definition.comment.end.verilog
#     ^ source.verilog meta.module.verilog
#      ^^ source.verilog meta.module.verilog comment.line.double-slash.verilog punctuation.definition.comment.verilog
>
>) //
#^ source.verilog meta.module.verilog punctuation.definition.ports.end.verilog
# ^ source.verilog meta.module.verilog
#  ^^ source.verilog meta.module.verilog comment.line.double-slash.verilog punctuation.definition.comment.verilog
>;
#^ source.verilog meta.module.verilog punctuation.section.module.begin.verilog
>endmodule
#^^^^^^^^^ source.verilog meta.module.verilog storage.type.module.verilog
>
>module complex_ports ({c, d}, .e(f));
#^^^^^^ source.verilog meta.module.verilog storage.type.module.verilog
#      ^ source.verilog meta.module.verilog
#       ^^^^^^^^^^^^^ source.verilog meta.module.verilog entity.name.type.module.verilog
#                    ^ source.verilog meta.module.verilog
#                     ^ source.verilog meta.module.verilog punctuation.definition.ports.begin.verilog
#                      ^ source.verilog meta.module.verilog
#                       ^ source.verilog meta.module.verilog variable.other.readwrite.verilog
#                        ^ source.verilog meta.module.verilog punctuation.separator.comma.verilog
#                         ^ source.verilog meta.module.verilog
#                          ^ source.verilog meta.module.verilog variable.other.readwrite.verilog
#                           ^ source.verilog meta.module.verilog
#                            ^ source.verilog meta.module.verilog punctuation.separator.comma.verilog
#                             ^ source.verilog meta.module.verilog
#                              ^ source.verilog meta.module.verilog punctuation.accessor.port.verilog
#                               ^ source.verilog meta.module.verilog variable.other.readwrite.verilog
#                                ^ source.verilog meta.module.verilog
#                                 ^ source.verilog meta.module.verilog variable.other.readwrite.verilog
#                                  ^ source.verilog meta.module.verilog
#                                   ^ source.verilog meta.module.verilog punctuation.definition.ports.end.verilog
#                                    ^ source.verilog meta.module.verilog punctuation.section.module.begin.verilog
>endmodule
#^^^^^^^^^ source.verilog meta.module.verilog storage.type.module.verilog
>
>module renamed_concat (.a({b, c}), f, .g(h[1]));
#^^^^^^ source.verilog meta.module.verilog storage.type.module.verilog
#      ^ source.verilog meta.module.verilog
#       ^^^^^^^^^^^^^^ source.verilog meta.module.verilog entity.name.type.module.verilog
#                     ^ source.verilog meta.module.verilog
#                      ^ source.verilog meta.module.verilog punctuation.definition.ports.begin.verilog
#                       ^ source.verilog meta.module.verilog punctuation.accessor.port.verilog
#                        ^ source.verilog meta.module.verilog variable.other.readwrite.verilog
#                         ^ source.verilog meta.module.verilog
#                          ^ source.verilog meta.module.verilog
#                           ^ source.verilog meta.module.verilog variable.other.readwrite.verilog
#                            ^ source.verilog meta.module.verilog punctuation.separator.comma.verilog
#                             ^ source.verilog meta.module.verilog
#                              ^ source.verilog meta.module.verilog variable.other.readwrite.verilog
#                               ^ source.verilog meta.module.verilog
#                                ^ source.verilog meta.module.verilog
#                                 ^ source.verilog meta.module.verilog punctuation.separator.comma.verilog
#                                  ^ source.verilog meta.module.verilog
#                                   ^ source.verilog meta.module.verilog variable.other.readwrite.verilog
#                                    ^ source.verilog meta.module.verilog punctuation.separator.comma.verilog
#                                     ^ source.verilog meta.module.verilog
#                                      ^ source.verilog meta.module.verilog punctuation.accessor.port.verilog
#                                       ^ source.verilog meta.module.verilog variable.other.readwrite.verilog
#                                        ^ source.verilog meta.module.verilog
#                                         ^ source.verilog meta.module.verilog variable.other.readwrite.verilog
#                                          ^ source.verilog meta.module.verilog meta.item-aceess.verilog meta.item-index.verilog punctuation.definition.begin.bracket.verilog
#                                           ^ source.verilog meta.module.verilog meta.item-aceess.verilog meta.item-index.verilog constant.numeric.dec.verilog
#                                            ^ source.verilog meta.module.verilog meta.item-aceess.verilog punctuation.definition.end.bracket.verilog
#                                             ^ source.verilog meta.module.verilog
#                                              ^ source.verilog meta.module.verilog punctuation.definition.ports.end.verilog
#                                               ^ source.verilog meta.module.verilog punctuation.section.module.begin.verilog
>endmodule
#^^^^^^^^^ source.verilog meta.module.verilog storage.type.module.verilog